package.json

```diff
@@ -26,6 +26,8 @@
     "eslint-plugin-sonarjs": "^0.18.0",
     "eslint-plugin-vue": "^8.7.1",
     "prettier": "^2.6.2",
-    "vue-eslint-parser": "^8.3.0"
+    "vue-eslint-parser": "^8.3.0",
```

Comment:
Pin the dependency, by freezing the dependencies we want to achieve repeatable deployment and make sure that every developer is testing on the very same codebase. Not with a range of possible packages